examples/lcg_expensive_LMO.jl

```diff
@@ -132,13 +139,34 @@ x, v, primal, dual_gap, trajectoryBCG = FrankWolfe.blended_conditional_gradient(
     line_search=FrankWolfe.adaptive,
     print_iter=k / 10,
     linesearch_tol=1e-9,
+    epsilon=target_accuracy,
     emphasis=FrankWolfe.memory,
     trajectory=true,
     verbose=true,
 );
 
 
-data = [trajectoryFW, trajectoryLCG, trajectoryBLCG, trajectoryLAFW, trajectoryBCG]
-label = ["FW", "L-CG", "BL-CG", "L-AFW", "BCG"]
+# BCG run (reference optimum)
+
+x0 = copy(x00)
+
+x, v, primal, dual_gap, trajectoryBCG_ref = FrankWolfe.blended_conditional_gradient(
+    x -> cf(x, xp),
+    (str, x) -> cgrad!(str, x, xp),
+    lmo,
+    x0,
+    max_iteration= 2 * k,
+    line_search=FrankWolfe.adaptive,
+    print_iter=k / 10,
+    epsilon=target_accuracy / 10.0,
+    linesearch_tol=1e-9,
+    emphasis=FrankWolfe.memory,
+    trajectory=true,
+    verbose=true,
+);
+
+open("lcg_expensive_data.json", "w") do f
+    write(f, JSON.json((FW=trajectoryFW, LCG=trajectoryLCG, BLCG = trajectoryBLCG, LAFW = trajectoryLAFW, BCG = trajectoryBCG, reference_BCG_primal = primal)))
+end
 
 FrankWolfe.plot_trajectories(data, label, xscalelog=true)
```

Comment:
you removed the data and label above, should this last line be adapted too?

Reply:
Yes, you're right.